resdk/tests/unit/test_resolwe.py

```diff
@@ -443,7 +450,8 @@ def setUp(self, auth_mock):  # pylint: disable=arguments-differ
     def test_bad_url(self, requests_mock):
         requests_mock.post = MagicMock(side_effect=[requests.exceptions.ConnectionError()])
 
-        with six.assertRaisesRegex(self, ValueError, r"Server not accessible on .*. Wrong url?"):
+        with six.assertRaisesRegex(self, ValueError,
+                                   'Server not accessible on www.abc.com. Wrong url?'):
```

Comment:
We wish to check this exact string (not the regex), because this exact url is used in this test case.